sleap/gui/commands.py

```diff
@@ -2984,6 +2985,7 @@ def set_visible_nodes(
         copy_instance: Optional[Union[Instance, PredictedInstance]],
         new_instance: Instance,
         mark_complete: bool,
+        location: Optional[QtCore.QPoint],
```

Comment:
When the type can be optional, I like to add a default value of `None` so that we can call this method without needing to specify the location:
```suggestion
        location: Optional[QtCore.QPoint] = None,
```